gnocchi/rest/aggregates/api.py

```diff
@@ -394,16 +456,14 @@ def truncate_measures(self):
                 self.truncate_measure(measure)
 
     def truncate_measure(self, measures_list):
-        if measures_list and len(measures_list) == 1:
+        if not measures_list:
             return
-        measures_sum = 0
-        for measure in measures_list[:-1]:
-            measure_new_val = (measure.value *
-                               measure.usage_coefficient)
-            measures_sum += measure_new_val
+
+        for measure in measures_list:
+            measure_new_val = (measure.value * measure.usage_coefficient)
+            LOG.debug('Trucating measure [%s] value to [%s].',
+                      measure.__dict__, measure_new_val)
```

Comment:
sorry, it's not clear to me how this change relates to your requirement to filter resources?

Reply:
While troubleshooting the problem to filter the resources, we were having a lot of issues to understand what was going on due to the lack of logs. Therefore, we added logs in a few places to helps us understand what was going on. It does not help fixing the issues per se, but it helped us to understand where the slowness was.

Reply:
While troubleshooting the problem to filter the resources, we were having a lot of issues to understand what was going on due to the lack of logs. Therefore, we added logs in a few places to helps us understand what was going on. It does not help fixing the issues per se, but it helped us to understand where the slowness was.

Reply:
oh i meant the change to measure_sum logic, not why you added the log. tbh, i don't remember why we ahve this measure_sum logic

Reply:
Ah, I see. This is something that was added when the implementation for the partial accounting of attributes was introduced (https://github.com/gnocchixyz/gnocchi/pull/1059). While working on this patch here, we were navigating the code, and trying to figure out what was going on. Therefore, we took the opportunity to add logs in the execution flows that were in need of some. 

As a consequence of that, we can save some operators going bald due to the lack of information for a troubleshooting.